neo/Prompt/Commands/tests/test_config_commands.py

```diff
@@ -198,44 +142,6 @@ def test_config_maxpeers(self):
             self.assertFalse(res)
             self.assertIn("Please supply a positive integer for maxpeers", mock_print.getvalue())
 
-        # test if the new maxpeers < settings.CONNECTED_PEER_MAX
-        # first make sure we have a predictable state
-        NodeLeader.Instance().Reset()
-        leader = NodeLeader.Instance()
-        addr1 = Address("127.0.0.1:20333")
-        addr2 = Address("127.0.0.1:20334")
-        leader.ADDRS = [addr1, addr2]
-        leader.DEAD_ADDRS = [Address("127.0.0.1:20335")]
-        test_node = NeoNode()
-        test_node.host = "127.0.0.1"
-        test_node.port = 20333
-        test_node.address = Address("127.0.0.1:20333")
-        test_node2 = NeoNode()
-        test_node2.host = "127.0.0.1"
-        test_node2.port = 20333
-        test_node2.address = Address("127.0.0.1:20334")
-        leader.Peers = [test_node, test_node2]
-
-        with patch("neo.Network.NeoNode.NeoNode.Disconnect") as mock_disconnect:
-            # first test if the number of connected peers !< new maxpeers
-            with patch('sys.stdout', new=StringIO()) as mock_print:
-                args = ['maxpeers', "4"]
-                res = CommandConfig().execute(args)
-                self.assertTrue(res)
-                self.assertEqual(len(leader.Peers), 2)
-                self.assertFalse(mock_disconnect.called)
-                self.assertIn(f"Maxpeers set to {settings.CONNECTED_PEER_MAX}", mock_print.getvalue())
-
-            # now test if the number of connected peers < new maxpeers
-            with patch('sys.stdout', new=StringIO()) as mock_print:
-                args = ['maxpeers', "1"]
-                res = CommandConfig().execute(args)
-                self.assertTrue(res)
-                self.assertEqual(len(leader.Peers), 1)
-                self.assertEqual(leader.Peers[0].address, test_node.address)
-                self.assertTrue(mock_disconnect.called)
-                self.assertIn(f"Maxpeers set to {settings.CONNECTED_PEER_MAX}", mock_print.getvalue())
-

```

Comment:
I saw that the maxpeers command was refactored but the logic added in https://github.com/CityOfZion/neo-python/pull/878 is still there so why is this test removed?

Reply:
Also, I noticed that there is now a `minpeers` attribute but no associated `config minpeers` or tests. Is this something I could/should add?

Reply:
This `asyncio` branch no longer has a `NodeLeader` class so this logic can't work and has been refactored (wait for new asyncio PR)

Reply:
Okay, I am still a little confused. Does this mean that you already have the new code but are just waiting to commit in a separate PR?